docs/src/understand/thin-edge-json.md

```diff
@@ -1,52 +1,107 @@
+| `tedge/alarms/${severity}/${alarm-type}`             | Alarms related to the main device                                      |
+| `tedge/alarms/${severity}/${alarm-type}/${child-id}` | Alarms related to the child device named `${child-id}`                 |
+| `tedge/errors`                                       | Error messages emitted by `thin-edge.io` while processing measurements |
+
+### Sending Telemetry Data to the cloud
+
+The `thin-edge.io` framework allows users forward telemetry data generated and published to one of the
+`tedge/#` MQTT topics from the thin-edge device to any IoT cloud provider that it is connected to,
+with the help of a *mapper* component designed for that cloud.
+The responsibility of a mapper is to subscribe to the `tedge/#` topic to receive all incoming measurements, events and alarms
+represented in the cloud vendor neutral Thin Edge JSON format, to a format that the connected cloud understands.
+Refer to [Cloud Message Mapper Architecture](./tedge-mapper.md) for more details on the mapper component.
+
+## Measurements
+
+*Measurements* carry values from physical sensors, the domain application or monitored processes.
+For instance:
+- voltage and current of an electricity meter
+- state of the manufacturing control process
+- free disk space on the device
+
+Thin Edge JSON can be used to represent single-valued measurements, multi-valued measurements
 or a combination of both along with some auxiliary data like the timestamp at which the measurement(s) was generated.
 
-## Single-valued measurements
+### Single-valued measurements
 
 Simple single-valued measurements like temperature or pressure measurement with a single value can be expressed as follows:
 
-```json
-{
+```sh te2mqtt
+tedge mqtt pub /tedge/measurements '{
   "temperature": 25
-}
+}'
 ```
 
 where the key represents the measurement type, and the value represents the measurement value.
 The keys can only have alphanumeric characters, and the "_" (underscore) character but must not start with an underscore.
 The values can only be numeric.
 String, Boolean or other JSON object values are not allowed.
 
-## Multi-valued measurements
+### Multi-valued measurements
 
 A multi-valued measurement is a measurement that is comprised of multiple values. Here is the representation of a
 `three_phase_current` measurement that consists of `L1`, `L2` and `L3` values, representing the current on each phase:
 
-```json
-{
+```sh te2mqtt
+tedge mqtt pub /tedge/measurements '{
   "three_phase_current": {
     "L1": 9.5,
     "L2": 10.3,
     "L3": 8.8
   }
-}
+}'
 ```
 
 where the key is the top-level measurement type and value is a JSON object having further key-value pairs 
 representing each aspect of the multi-valued measurement.
 Only one level of nesting is allowed, meaning the values of the measurement keys at the inner level can only be numeric values.
 For example, a multi-level measurement as follows is NOT valid: 
 
```

Comment:
I would add a more explicit sub heading (bold) text to highlight that it is an invalid example, as it is easy to overlook the sentence mentioning that it is an invalid example:

```suggestion

❌ **Example: Invalid measurement due to nesting > 2 levels**
```

Reply:
Added